subtests/docker_cli/systemd/docker-build.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=docker-autotest systemd build image subtest. If you see this file on a live system it means something has failed very badly
+
+[Service]
+Type=oneshot
+Environment=NAME=set-by-the-test-from-systemd.ini
```

Comment:
Why two different strings? I like `to-be-set-by-test` (below), it's short and descriptive. Why not use that for `NAME=` also? Imagine yourself as the maintenance programmer. You see this:

    NAME=to-be-set-by-test
    DOCKERFILEDIR=to-be-set-by-test

Your first thought is, aha, those are identical values in contexts where they should be different, that clearly means that I should look elsewhere, in one other place, to see how they're set. But if you see:

    NAME=one-thing
    DOCKERFILEDIR=another

...you might wonder: why are those different? What is the reason for using one replaceable value in one place, and another in the other?